pulsar-io/kafka/pom.xml

```diff
@@ -54,6 +54,31 @@
       <version>${kafka-client.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>io.confluent</groupId>
+      <artifactId>kafka-schema-registry</artifactId>
+      <version>${kafka.confluent.schemaregistryclient.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>io.confluent</groupId>
+      <artifactId>kafka-avro-serializer</artifactId>
+      <version>${kafka.confluent.avroserializer.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>io.confluent</groupId>
+      <artifactId>kafka-streams-avro-serde</artifactId>
+      <version>${kafka.confluent.avroserializer.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.apache.pulsar</groupId>
+      <artifactId>pulsar-client-original</artifactId>
+      <version>${project.version}</version>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
Is the pulsar-client-original dependency needed in the compile scope? the problem is that it will pull a lot of dependencies to the .nar files that are built. Perhaps `provided` scope would be fine? However I think that `provided` scope isn't transient and therefore it might not have impact in this library.

Reply:
I have moved this to provided, good catch